pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: shelf_web_socket
-version: 0.2.1
+version: 0.2.2
```

Comment:
Also need to bump the min SDK to a 1.23 (I think) – A version that has generic method support

Reply:
Done.